### PR TITLE
done

### PR DESCRIPTION
**Pull request by fhoman**

### CODE REVIEW

File: starter-code/css/main.css [hunk 1]
```diff
@@ -6,3 +6,278 @@ Green: #00B172
...
+    
+display: inline;
+font-size: 25px;
+padding: 0px 10px 0px 10px;
+  
+}
+
+nav div.logo {
+
+background-image: url('/starter-code/images/spotify-logo.png');
```
Comment (LloydC) [inline starter-code/css/main.css]: You need to update the url path to '../images/spotify-logo.png'

File: starter-code/css/main.css [hunk 2]
```diff
@@ -6,3 +6,278 @@ Green: #00B172
...
+
+
+/* Header image section*/
+
+
+
+
+section.header-image {
+
+background-image: url('/starter-code/images/landing.jpg');
```
Comment (LloydC) [inline starter-code/css/main.css]: You need to update the url path to '../images/landing.jpg'

File: starter-code/css/main.css [hunk 3]
```diff
@@ -6,3 +6,278 @@ Green: #00B172
...
+margin-left: auto;
+margin-right: auto;
+
+
+}
+
+
+div.loudspeaker-icon {
+
+background-image: url('/starter-code/images/high-quality-icon.png');
```
Comment (LloydC) [inline starter-code/css/main.css]: Same as mentioned above

File: starter-code/css/main.css [hunk 4]
```diff
@@ -6,3 +6,278 @@ Green: #00B172
...
+height: 100px;
+width: 100px;
+background-repeat: no-repeat;
+background-size: cover;
+
+}
+
+div.music-icon {
+
+background-image: url('/starter-code/images/music-icon.png');
```
Comment (LloydC) [inline starter-code/css/main.css]: Same as mentioned above

File: starter-code/css/main.css [hunk 5]
```diff
@@ -6,3 +6,278 @@ Green: #00B172
...
+height: 100px;
+width: 100px;
+background-repeat: no-repeat;
+background-size: cover;
+    
+}
+
+div.stream-icon {
+
+background-image: url('/starter-code/images/devices-icon.png');
```
Comment (LloydC) [inline starter-code/css/main.css]: Same as mentioned above

File: starter-code/css/main.css [hunk 6]
```diff
@@ -6,3 +6,278 @@ Green: #00B172
...
+
+position: relative;
+
+
+    
+    }
+
+div.logo-white {
+
+background-image: url('/starter-code/images/spotify-icon-white.png');
```
Comment (LloydC) [inline starter-code/css/main.css]: Same as mentioned above

File: starter-code/css/main.css [hunk 7]
```diff
@@ -6,3 +6,278 @@ Green: #00B172
...
+
+padding-right: 90px;
+margin-right: 120px;
+
+}
+
+
+div.app-image {
+
+    background-image: url('/starter-code/images/spotify-app.jpg');
```
Comment (LloydC) [inline starter-code/css/main.css]: Same as mentioned above

File: starter-code/css/main.css [hunk 8]
```diff
@@ -6,3 +6,278 @@ Green: #00B172
...
+
+
+div.usp-box div {
+
+margin-left: auto;
+margin-right: auto;
+
+
+}
+
```
Comment (LloydC) [inline starter-code/css/main.css]: I noticed that your classes '.loudspeaker-icon','.music-icon' and '.stream-icon' have 4 identical CSS properties. I would suggest maybe creating a single class '.icon-model' which can be used on as many icons as you need, you get less code and better structure.

Comment (fhoman) [reply starter-code/css/main.css]: Thanks, i adjusted the paths and made a general icon class

**Pull request closed without merging**